<?xml version="1.0"?>
<launch>

  <node pkg="tensorflow_object_detector"
        type="tensorflow_object_detector"
        name="tensorflow_object_detector"
        output="screen"
        launch-prefix=""> <!-- use "gdb \-\-args" for debugging with gdb -->
    <param name="graph_path" value="$(find tensorflow_object_detector)/config/ssd_mobilenet_v1_coco.pb"/> 
    <param name="labels_path" value="$(find tensorflow_object_detector)/config/mscoco_labels.txt"/>
    <param name="score_threshold" value="0.3"/> 
    <param name="always_output_image" value="true"/> 
  
    <remap from="image_in" to="/vica_robot/sensor/kinect/rgb/image_raw"/>
    <remap from="image_out" to="/image_out"/>
  </node>

  <node pkg="image_view" type="image_view" name="viewer">
    <remap from="image" to="/image_out"/>
  </node>
</launch>
